### ISSUE TITLE
Suggestion: Move pkg to another git repository or storage

### ISSUE THREAD

**Issue by hajimehoshi** [ebiten-playground/ebiten-playground.github.io#10]
As pkg is too big and its history doesn't have to be managed, I suggest moving pkg directory to another git repository or storage like GCS or AWS.

Actually,  `go get -u github.com/gopherjs/gopherjs.github.io` that has a similar pkg directory unnecessarily takes very long.

**Comment by j7b** [comment]
Sure, could also package to npm and use the npm cdn's.

**Comment by hajimehoshi** [comment]
.a files are just binaries, so I'm not sure they can be a npm package.

**Comment by j7b** [comment]
Maybe a good approach is to build jsplayground.js and pkg to a gh-pages branch somewhere (here or mine) committed with --force to squash it, I think travis supports that.

**Comment by hajimehoshi** [comment]
My suggestion is to create a completely different GitHub repository. gh-pages branch is a little confusing as my experience. Actually example js files for Ebiten are in https://github.com/hajimehoshi/ebiten.pagestorage .

**Comment by j7b** [comment]
I need to figure out the ci story for jsplayground and add a couple things, then the artifacts and additionals can live somewhere else pretty easily.

**Comment by hajimehoshi** [comment]
For testing, an argument or a command line or an environment variable to specify pkg location would work in CI. I guess you want to do integrated test from generating pkg to the playground, right?

**Comment by j7b** [comment]
yep so far testing at https://github.com/j7b/citesting and pretty happy with how that .travis.yml works, was thinking to make repo ebiten-playground/js and putting jsplayground.js and pkg in it in gh-pages and you could put hook on ebiten repo so it'll rebuild on push or release, probably push would be best

**Comment by hajimehoshi** [comment]
Great!

One question (probably I've already asked): I feel like `gopherjs install github.com/j7b/jsplayground` should be `gopherjs build -m -o jsplayground.js github.com/j7b/jsplayground` and moving the file.
